### ISSUE TITLE
Support for <input>

### ISSUE THREAD

**Issue by dertuxmalwieder** [cheunghy/jquery.colorfy#11]
Is there a way to make colorfy work with standard `input` one-line text fields?

**Comment by victorteokw** [comment]
It should just work for input. See example [here](http://cheunghy.github.io/jquery.colorfy/)

**Comment by dertuxmalwieder** [comment]
Oh, thanks.

In this case: It's broken.

![Screenshot, Firefox, Windows 10](http://i.imgur.com/0enEKtV.png)

Can I help?

**Comment by victorteokw** [comment]
I've pushed a new commit, which should fix this issue. Try [here](http://cheunghy.github.io/jquery.colorfy/)

**Comment by dertuxmalwieder** [comment]
Still looks the same to me ...

**Comment by victorteokw** [comment]
What browser and system you are using?

**Comment by dertuxmalwieder** [comment]
Firefox (a broken look is _not_ a part of the known Firefox bugs yet, right?) on Windows 10.

`overflow:auto` works for me here.

**Comment by victorteokw** [comment]
So you have workaround for it, with `overflow:auto`, right?

**Comment by dertuxmalwieder** [comment]
Yes, but (of course) this should be a part of the `colorfy` plug-in IMO.

Also, the problems occurs again when the text in the `input` field becomes long enough.

**Comment by victorteokw** [comment]
Yes, it should be a part of `colorfy`.
